### PR TITLE
[DinoMod] bear armor

### LINKED ISSUE #60097: Can't equip elephant armor on pet dinos
### Describe the bug

Pet dinos with elephant bodytype can't be equipped with chitin elephant armor. 

### Steps to reproduce

1. Tame mon_parasaurolophus, with bodytype "elephant".
2. Craft chitin elephant armor and wield it: 
image
3. Select to equip the dino with armor: 
image
4. Get this error: 
image

### Expected behavior

Dino with bodytype elephant does accept the elephant chitin armor.

### Screenshots

_No response_

### Versions and configuration

- OS: Linux
    - OS Version: Distributor ID: Ubuntu; Description: Ubuntu 21.10; Release: 21.10; Codename: impish; 
- Game Version: df06cbf [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Innawood [innawood],
    DinoMod [DinoMod],
    Megafauna [megafauna],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills]
]

### Additional context

Please ignore the smartphone in my inventory, playing as an innawoods char :) That's another bug.

### PR DESCRIPTION
**Pull request by ElizabethThorner**
#### Summary
Content "Added bear body type armors to DinoMod"

#### Purpose of change

Fixes  #60097

#### Describe the solution

I created pets_bear.json, pets_bear_armor.json, and I added code to recipes.json. All of these were nearly identical to the code for the elephant and bird body type armor variants. I also added a line to dinosaurs.json to try to ensure that the parasaurolophus had the bear bodytype, since apparently it showed up as elephant in game.

#### Describe alternatives you've considered

I didn't really consider alternative solutions because I was provided a template to work with.

#### Testing

There's no reason why it shouldn't work, since the code is nearly the same as the code adding the bird and elephant body type armor variants, but I honestly struggled to figure out how to get the game running and get these armors in the first place so I wasn't able to test it.

### CODE REVIEW

File: data/mods/DinoMod/monsters/dinosaur.json [hunk 1]
```diff
@@ -1673,6 +1673,7 @@
     "name": { "str_sp": "parasaurolophus" },
     "copy-from": "mon_hadrosaurus",
     "//": "When large weights and volumes are fixed this should be 4000 L and 4000 kg",
+    "bodytype": "bear",
```
Comment (LyleSY) [inline data/mods/DinoMod/monsters/dinosaur.json:1676]: Huh, this should be automagically handled by the copy-from line. It was an elephant in testing for you?

File: data/mods/DinoMod/items/pets_bear_armor.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    "volume": "150 L",
+    "material_thickness": 6
+  },
+  {
+    "type": "PET_ARMOR",
+    "id": "rubber_armor_bear",
+    "copy-from": "bear_armor",
+    "color": "green",
+    "name": { "str": "bear rain sheet" },
+    "description": "A thin plastic covering, adapted to protect an bear from acid zombie splashes and similar caustic substances.  You could put this on a friendly bear.",
```
Comment (LyleSY) [inline data/mods/DinoMod/items/pets_bear_armor.json:131]:
```suggestion
    "description": "A thin plastic covering, adapted to protect a bear from acid zombie splashes and similar caustic substances.  You could put this on a friendly bear.",
```

**Review by LyleSY — COMMENTED** [review]
Overall really good, love the caparison stuff, and happy to see the holiday sweater in there. The copy-from feature should make the change to the dinosaur file unneeded. Should. Thanks!

File: data/mods/DinoMod/monsters/dinosaur.json (hunk 1 above)
Comment (LyleSY) [inline data/mods/DinoMod/monsters/dinosaur.json:1676]:
```suggestion
```
In theory this shouldn't be needed

Comment (ZeldaGaladriel) [reply data/mods/DinoMod/monsters/dinosaur.json:1676]: Yeah you're probably right, I was just a little confused by the discussion in the issue with OP saying that the dinosaur had the elephant body type.

**Review by LyleSY — APPROVED** [review]
Great start, hope you make more contributions

**Pull request merged**